native/GitExtensionsShellEx/Generated/GitExtensionsShellEx.h

```diff
@@ -0,0 +1,224 @@
+
```

Comment:
Why separating native?
What "standard" is this based on?

Reply:
None. These projects are C++, and are not part of the main solution. Having them in the same folder as the C# projects, which are part of the solution, feels confusing. These projects are built via native.proj too (I should move it from eng/ here). So, the name "native" stemmed from there.

Do you have better suggestions?

Reply:
move below src/, as "top" level project feels natural to me
Alternative is to add yet another layer with app/, shellext/ below src/

Reply:
No strong feelings: I find this `/src/native` cleaner and more meaningful (i.e. better organized) but maybe a little less convenient.
I think I have a little preference for `/src/native`...